apps/supernova/src/components/silences/SilenceNew.jsx

```diff
@@ -78,6 +80,10 @@ const SilenceNew = ({ alert, size, variant }) => {
       createdBy: authData?.parsed?.fullName,
       matchers: setupMatchers(alert?.labels, excludedLabels, enrichedLabels),
     })
+
+    if (authData?.parsed?.fullName === "anonymous") {
```

Comment:
Same here. No need to set again this information in each component if it is set in the store.